collect.py

```diff
@@ -241,19 +240,30 @@ def run_tests(firefox_driver, chrome_driver):
     chrome_driver.quit()
 
 
+NO_OF_THREADS = 5
```

Comment:
Can you move this at the top of the file? We want all global variables to be there.

Reply:
Nit: And also rename it `THREADS_NUM` or `MAX_THREADS`